### PR TITLE
Enable Learning Objectives in Chapter Outline

### PR DESCRIPTION
**Pull request by helenemccarron**
Specifically in Physics but applicable everywhere as well. Referencing #185.

### CODE REVIEW

File: rulesets/mixins/_compose.scss [hunk 1]
```diff
@@ -128,7 +128,7 @@
 /// - collated sections are moved into their corresponding "bucket"
 ///
 /// @arg {Map} $page - contains `name`, `source`, and optionally `compoundComposite` and `specialPageType`
-/// @arg {string} $sectionHeaderNode - Curently Unused (because of typo)
+/// @arg {string} $sectionHeaderNode - Currently Unused (because of typo)
```
Comment (tomjw64) [inline rulesets/mixins/_compose.scss]: 🥇

Comment (helenemccarron) [inline rulesets/mixins/_compose.scss]: Hehehe

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# :+1::tada: Thanks for Contributing! :tada::+1:
```
Comment (philschatz) [inline CONTRIBUTING.md]: How did this file get into the PR diff?

File: rulesets/books/_generator.scss [hunk 3]
```diff
@@ -155,9 +155,10 @@ $_symbolsSectionTitle: "Symbols";
 
   //create a chapter outline
   $outlineTitle: map-get($Config_PartType_Chapter, outlineTitle);
+  $hasLearningObjectives: map_get($Config_PartType_Chapter, hasLearningObjectives);
```
Comment (philschatz) [inline rulesets/books/_generator.scss]: Can you update the docs in https://github.com/Connexions/cnx-recipes/tree/master/rulesets#custompart (I probably forgot another field in a previous PR...

Comment (helenemccarron) [inline rulesets/books/_generator.scss]: Made the change but this comment doesn't seem to have picked on it.

File: rulesets/mixins/_compose.scss (hunk 1 above)
Comment (kerwinso) [inline rulesets/mixins/_compose.scss]: think about the irony

**Comment by kerwinso** [comment]
Do not merge without checking with devs first, this may be dependent on #273 being merged first.

**Pull request merged**